test/unit/math/rev/mat/prob/ordered_logistic_test.cpp

```diff
@@ -296,3 +296,34 @@ TEST(ProbDistributionsOrdLog, intErrors) {
   EXPECT_FLOAT_EQ(std_c_v[3][1].adj(), 0.0);
   EXPECT_FLOAT_EQ(std_c_v[3][2].adj(), -0.70889017256612);
 }
+
+TEST(ProbDistributionsOrdLog, vv_vec_y1) {
+  using stan::math::ordered_logistic_lpmf;
+  using stan::math::var;
+  using stan::math::vector_d;
+  using stan::math::vector_v;
+
+  std::vector<int> y{1, 1, 1, 1};
+
+  vector_v lam_v(4);
+  lam_v << -2.95, -1.68, 0.96, 2.68;
+
+  vector_v c_v(3);
+  c_v << -2.68, -1.53, 0.46;
+
+  vector_d c_adj(4);
+  c_adj << 0.4329070950345456674650, 0.7310585786300048792511,
```

Comment:
[optional]

Where did you get these values?  I would prefer finite difference tests because they can be made very generic and I know what's being tested.  As long as the values are tested properly at the `double` level, we can use finite diffs to test all of our autodiff.